radboudumc-theme/sass/_typography.scss

```diff
@@ -33,11 +55,64 @@ h1.display-title {
   margin-bottom: 1em;
 }
 
-
 // First header on each page
 .page-title {
   font-size: 3rem;
+  font-weight: 800;
+
+  margin-bottom: 1.5rem;
+  font-size: 2rem;
+}
+
+p a { 
+  font-weight: 800;
+}
+
+h3 a {
   font-weight: 300;
```

Comment:
Is this required? Given the default of 300?

Reply:
This is here because all links are now bold inside a paragraph. However making h3 links bold as well is too much in my opinion.